.github/workflows/test-pro-rules.jsonnet

```diff
@@ -0,0 +1,114 @@
+// This workflow ...
+
+local actions = import 'libs/actions.libsonnet';
+local semgrep = import 'libs/semgrep.libsonnet';
+
+// ----------------------------------------------------------------------------
+// The job
+// ----------------------------------------------------------------------------
+
+local setup_docker_tag_job = {
+  name: 'Set up Docker tag based on if this is a pull request',
+  'runs-on': 'ubuntu-22.04',
+  outputs:
+    {
+      'docker-tag': '${{ steps.setup-docker-tag.outputs.docker-tag }}'
+    },
+  steps: [
+    {
+      name: 'Setup Docker Tag',
+      id: 'setup-docker-tag',
+      run: |||
+        echo "Github event is ${{ github.event_name }}"
+        if [ "${{ github.event_name }}" = "pull_request" ]; then
+          echo "docker-tag=pr-${{ github.event.pull_request.number }}" >> "$GITHUB_OUTPUT"
+          echo "Setting docker tag to current pull request number"
+        else
+          echo "docker-tag=develop" >> "$GITHUB_OUTPUT"
+          echo "Setting dry-run to develop"
+        fi
+      |||
+    }
+  ]
+};
+
+local test_pro_rules_job = {
+  'runs-on': 'ubuntu-latest',
+  needs: 'setup-docker-tag',
+  // Switching to Ubuntu here because Alpine does not provide easily 'gh'
```

Comment:
Do not put back this long comment. Just refer to check-semgrep-pro.jsonnet for more info